src/components/pages/BookPage/starRate.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+function CreateStars({ rate }) {
+  let stars = [];
+  let firstDigit = rate - parseInt(rate);
+  let secondDigit = firstDigit.toString().split('.')[1];
+  let createStars = () => {
+    for (let i = 1; i <= 5; i++) {
```

Comment:
refactor the magic number 5 into constant and put it at the top of the page

Reply:
i did not create starts zeena did. she sent  it and i copy it.

Reply:
i'll solve it